src/AzureFunctionPackage/Functions/GitHubAutoMergePRs/GitHubAutoMergePRs.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net46</TargetFramework>
+  </PropertyGroup>
+
+  <!--
+
+  The dependencies specified in `project.json` are the real ones.  These simply exist so that the generated dependency
+  graph (`project.assets.json`) can be copied as a build step.
```

Comment:
Do we have a plan to remove this duplication? Is this something we also invented or is the standard SDK?

Reply:
Azure Functions currently only support `project.json` scenarios, but since that's not something we can easily build and test locally due to the 2.x SDK completely removing `project.json` support, this was the next best thing.